content/blog/2023-10-31-cis-benchmarks-update.md

```diff
@@ -0,0 +1,76 @@
+| 8.8 - Management of technical vulnerabilities | 7.3 - Perform Automated Operating System Patch Management                   | 1.2 - Configure Software Updates     |
+| 8.2 - Privileged access rights                | 5.4 - Restrict Administrator Privileges to Dedicated Administrator Accounts | 5.3 - Configure privilege escalation |
+
+## How can I test my servers for compliance?
+
+Many vulnerability scanners have the capability to scan a host for compliance to various CIS benchmarks.
+
+[OpenSCAP](https://wiki.almalinux.org/documentation/openscap-guide.html) is an open-source solution that we provide instructions and packages for.
+
+CIS have their own [CIS-CAT Pro](https://www.cisecurity.org/cybersecurity-tools/cis-cat-pro) which can be run locally via the commandline or remotely via GUI. CIS-CAT is tightly linked to the benchmarks so updates are published simultaneously.
+
+[Tenable](https://www.cisecurity.org/partner/tenable)'s Nessus Pro includes compliance audits against published standards and baselines (CIS, DISA STIGs) and across more than 15+ security frameworks (HIPAA, NIST 800-53). The dedicated compliance team comes with decades of combined experience and are an active part of industry working groups such as CIS. I recently worked with Tenable Research to improve the accuracy of the AlmaLinux OS 9 CIS audits.
+
+[Qualys](https://www.cisecurity.org/partner/qualys) Policy Compliance assesses various CIS benchmarks - including AlmaLinux, as well as prioritises the risk of the misconfiguration by mapping them to the MITRE ATT&CK framework and threats such as ransomware, with the ability to remediate misconfigurations to reduce risk and bring servers into compliance.
+
+## How can I security harden my servers?
+
+On my [github](https://github.com/sej7278/virt-installs) I have a few scripts for hardening various distro's including AlmaLinux, to CIS benchmarks, using some of my favourite technologies - Kickstart and libvirt, but also Ansible, cloud-init and Preseed.
+
+With CIS [SecureSuite](https://www.cisecurity.org/cis-hardened-images) membership you can get access to pre-hardended virtual machine images for most cloud providers, build-kits for hardening existing infrastructure and the CIS-CAT Pro tool for testing.
+
+Unlike some security frameworks which can be quite woolly and high-level, the CIS benchmarks are distribution-specific and very descriptive - they tell you exactly which commands to type or what to put in a config file to remediate non-compliances and how to retest. This is what most companies I've worked with find attractive - a sysadmin can do the hardening and a SOC analyst can measure compliance without hiring a consultancy to tell you what is meant by phrases like *"use sufficient cryptographic controls for the classification of data at rest and in transit"*.
```

Comment:
I've not heard the word woolly used in this context before and am not sure what it means for you. If it's too colloquial we should adjust it to a more common word, or maybe just add more context.

Reply:
Changed to "vague"